src/index.html

```diff
@@ -11,7 +11,164 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="fieldset">
+
+        <legend>Personal information</legend>
+
+        <label class="form-field" for="surname">
+          Surname:
+          <input
+            type="text"
+            id="surname"
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="form-field" for="name">
+          Name:
+          <input
+            type="text"
+            id="name"
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="form-field" for="age">
+          How old are you?
+          <input
+            type="number"
+            id="age"
+            min="1"
+            max="100"
+            value="12"
+          >
+        </label>
+
+        <label for="birthDate" class="form-field">
+          Full date of Birth:
+          <input type="date" id="birthDate">
+        </label>
+
+        <label class="form-field" for="term">I accept the term of the aggrement
```

Comment:
You should be consistent about your new lines - if you move the closing tag to a new line - you move the text too.
```suggestion
        <label class="form-field" for="term">
          I accept the term of the agreement
```